test/orm/declarative/test_typed_mapping.py

```diff
@@ -105,6 +105,8 @@ class _SomeDict2(TypedDict):
 _StrPep695: TypeAlias = Union[_SomeDict1, _SomeDict2]
```

Comment:
**Michael Bayer** (zzzeek) wrote:

weird, these were reversed?

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5260